src/zzz_od/application/zzz_application.py

```
@@ -36,5 +45,17 @@ def __init__(self, ctx: ZContext, app_id: str,
         )
 
     def handle_resume(self) -> None:
+        """恢复运行时激活游戏窗口。"""
         self.ctx.controller.active_window()
         Application.handle_resume(self)
+
+    def after_operation_done(self, result: OperationResult) -> None:
+        """应用成功时执行后置操作，再按最终结果更新记录和发送通知。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)